readme.md

```diff
@@ -29,6 +29,16 @@ To install an unoffical community maintained [Homebrew](https://brew.sh) [Cask](
 brew cask install sequel-ace
 ```
 
+### Moving saved connection list from Sequel Pro
+
+To move your quick connect list from Sequel Pro to Sequel Ace just copy the file
```

Comment:
Should we just add info and pointer to separate page that we have? https://sequel-ace.com/get-started/migrating-from-sequel-pro.html

Reply:
Well yes, this link would be useful, especially that it's not directly accessible from https://sequel-ace.com/, only through  FAQ, that is not immediately visible.

Reply:
Let's update this with something like `For more migration info, check [migration guide](https://sequel-ace.com/get-started/migrating-from-sequel-pro.html)` then? :)

Reply:
So maybe just 

### Moving saved connection list from Sequel Pro
And the link here

Reply:
Yeah something like that